Build failure: quicktemplate

### Nixpkgs version

- Unstable (25.05)

### Steps to reproduce

`nix-build -A quicktemplate`

### Can Hydra reproduce this build failure?

Yes, Hydra can reproduce this build failure.

### Link to Hydra build job

https://hydra.nixos.org/search?query=quicktemplate

### Relevant log output

```console
nix log /nix/store/734dgsbyswmyrakf1ww2b9x12w1rdmcv-quicktemplate-1.7.0.drv
Running phase: unpackPhase
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking source archive /nix/store/jyxvb3lpn586ykjfa1pz4cvhkaqz48rf-source
source root is source
Running phase: patchPhase
@nix { "action": "setPhase", "phase": "patchPhase" }
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
Running phase: configurePhase
@nix { "action": "setPhase", "phase": "configurePhase" }
Running phase: buildPhase
@nix { "action": "setPhase", "phase": "buildPhase" }
Building subPackage .
Building subPackage ./parser
Building subPackage ./qtc
Building subPackage ./tests
Running phase: checkPhase
@nix { "action": "setPhase", "phase": "checkPhase" }
--- FAIL: TestAppendJSONString (0.00s)
    jsonstring_test.go:44: unexpected result "\\u0000\\n\\r\\t\\u0008\\u000c\\\"\\\\". Expecting "\\u0000\\n\\r\\t\\b\\f\\\"\\\\". original string "\x00\n\r\t\b\f\"\\"
FAIL
FAIL    github.com/valyala/quicktemplate        0.001s
FAIL
```

### Additional context

_No response_

### System metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.12.17, NixOS, 25.05 (Warbler), 25.05pre-git`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.24.12`
 - nixpkgs: `/nix/store/xq5rfjj1z2r8yx338arajg5vwsxh1fri-source`



### Notify maintainers

@Madouura 
---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/qu/quicktemplate/package.nix
+++ b/pkgs/by-name/qu/quicktemplate/package.nix
@@ -3,16 +3,15 @@
   buildGoModule,
   fetchFromGitHub,
 }:
-
 buildGoModule rec {
   pname = "quicktemplate";
-  version = "1.7.0";
+  version = "1.8.0";
 
   src = fetchFromGitHub {
     owner = "valyala";
     repo = "quicktemplate";
     rev = "v${version}";
-    sha256 = "0xzsvhpllmzmyfg8sj1dpp02826j1plmyrdvqbwryzhf2ci33nqr";
+    sha256 = "cra3LZ3Yq0KNQErQ2q0bVSy7rOLKdSkIryIgQsNRBHw=";
   };
 
   vendorHash = null;

